source/core/ib_stencils.cc

```diff
@@ -0,0 +1,235 @@
+//
+// Created by lucka on 2021-05-21.
+//
+
+#include <core/ib_stencils.h>
+
+
+
+template <int dim>
+unsigned int
+IBStencils<dim>::nb_points(unsigned int order)
+{
+  // The number of points used in the stencil excluding the DOF is equal to the
+  // order.
+  unsigned int nb_points = order;
+  // In the case where the cell is used directly a=to to find the solution at
+  // the IB only one point is needed.
+  if (order > 4)
+    nb_points = 1;
+  return nb_points;
+}
+
+template <int dim>
+std::vector<double>
+IBStencils<dim>::coefficients(unsigned int order)
+{
+  // Initialize the coefficient vector
+  std::vector<double> coef;
```

Comment:
Can you resize here?
The size is order+1 except if order>4. You could set the size here for everyone once and for all.